lib/travis/cli/api_command.rb

```diff
@@ -108,7 +108,7 @@ def setup_enterprise
             endpoint
           end
           self.api_endpoint             = c[enterprise_name]
-          self.insecure                 = true if insecure.nil?
+          self.insecure                 = true unless insecure.nil?
```

Comment:
it failed a build by adding a test in https://github.com/travis-ci/travis.rb/pull/487/commits/e2fd23ed67d6bfe692b06eb71dd0db5b6f1b45b6 and fixed by https://github.com/travis-ci/travis.rb/pull/487/commits/619aa552d063d25e8997ab2758782ad70304dd5c